SteakBot.Core/EventHandlers/ReactionEventHandler.cs

```diff
@@ -11,9 +11,17 @@ public async Task HandleReactionAddedAsync(Cacheable<IUserMessage, ulong> arg1,
         {
             var channel = arg3.Channel;
             var message = await channel.GetMessageAsync(arg3.MessageId);
+            var url = message.GetJumpUrl();
 
-            await channel.SendMessageAsync($"{arg3.User.Value.Mention} reacted with {arg3.Emote} to a message by {message.Author.Username}"
-                                           + $" from {message.CreatedAt.LocalDateTime:HH:mm:ss} of {message.CreatedAt.LocalDateTime:dd.MM.yyyy}.");
+            var text = $"{arg3.User.Value.Mention} reacted with {arg3.Emote} to [a message]({url}) by {message.Author.Username}"
```

Comment:
Change var names to be more readable (text -> reactionMessage, embed -> embedMessage for example).